### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Mzuha**

### CODE REVIEW

File: src/main/resources/init_db.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+CREATE SCHEMA `cinema` DEFAULT CHARACTER SET utf8 ;
```
Comment (aaananas) [inline src/main/resources/init_db.sql:1]: you dont need to add script to create schema

**Review by aaananas — CHANGES_REQUESTED** [review]

File: src/main/resources/init_db.sql (hunk 1 above)
Comment (Mzuha) [reply src/main/resources/init_db.sql:1]: why?

**Review by aaananas — APPROVED** [review]

File: src/main/resources/hibernate.cfg.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <property name="connection.username">root</property>
+        <property name="connection.password">qwerty12345</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Actor"/>
+        <mapping class="mate.academy.hibernate.relations.model.Country"/>
+        <mapping class="mate.academy.hibernate.relations.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (KseniiaMakarova) [inline src/main/resources/hibernate.cfg.xml:15]: add an empty line to the end

File: src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java [hunk 3]
```diff
@@ -1,9 +1,16 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance = initSessionFactory();
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java:7]:
```suggestion
    private static final SessionFactory instance = initSessionFactory();
```

File: src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
+
+    private static SessionFactory initSessionFactory() {
+        return new Configuration().configure().buildSessionFactory();
+    }
+
     public static SessionFactory getSessionFactory() {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java:13]: please see the point about the constructor in the checklist

File: src/main/java/mate/academy/hibernate/relations/model/Movie.java [hunk 5]
```diff
@@ -2,10 +2,25 @@
...
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
 
+@Entity
 public class Movie implements Cloneable {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/hibernate/relations/model/Movie.java:15]: how are your tables called in the DB?
maybe we should add the `@Table` annotation for all entities?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**